File: app/src/main/java/amrabed/android/release/evaluation/about/AboutSection.java
package amrabed.android.release.evaluation.about;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.DialogFragment;

import amrabed.android.release.evaluation.R;

public class AboutSection extends DialogFragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        final TextView view = (TextView) inflater.inflate(R.layout.about, container, false);
        view.setText(R.string.about_content1);
        return view;
    }
}

File: app/src/main/java/amrabed/android/release/evaluation/about/AboutSection.kt
package amrabed.android.release.evaluation.about

import amrabed.android.release.evaluation.R
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.DialogFragment

class AboutSection : DialogFragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.about, container, false) as TextView
        view.setText(R.string.about_content1)
        return view
    }
}